### ISSUE TITLE
Code sign in error when installing using carthage

### ISSUE THREAD

**Issue by tbaranes** [PGSSoft/InAppPurchaseButton#2]
```
/usr/bin/xcrun xcodebuild -workspace path_to_project/Carthage/Checkouts/InAppPurchaseButton/InAppPurchaseButton.xcworkspace -scheme InAppPurchaseButton\ iOS -configuration Release -derivedDataPath path_to_library/Library/Caches/org.carthage.CarthageKit/DerivedData/InAppPurchaseButton/1.0.1 -sdk iphoneos ONLY_ACTIVE_ARCH=NO BITCODE_GENERATION_MODE=bitcode CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES clean build (launched in path_to_project/Carthage/Checkouts/InAppPurchaseButton)
```

**Comment by pwlkania** [comment]
@tbaranes It should be fixed in 1.0.2. Please check this out.

**Comment by tbaranes** [comment]
Working, thanks 👍
